docs/writing-tests/testdriver.md

```diff
@@ -174,6 +174,23 @@ await test_driver.set_permission({ name: "background-fetch" }, "denied");
 await test_driver.set_permission({ name: "push", userVisibleOnly: true }, "granted", true);
 ```
 
+### set_timezone
```

Comment:
The HTML Standard consistently uses "time zone" (not "timezone") so we should follow that here.

```suggestion
### set_time_zone
```

…and similar for the rest of the patch